BeeSwift/GoalViewController.swift

```diff
@@ -284,7 +287,70 @@ class GoalViewController: UIViewController,  UIScrollViewDelegate, DatapointTabl
         setValueTextField()
         updateInterfaceToMatchGoal()
     }
-
+    
```

Comment:
Is there somewhere outside `GoalViewController` it would make sense to extract this to?

Reply:
My next move would be just to refactor it into a private extension of GoalVC further down in this file.